bodhi/server/scripts/skopeo_lite.py

```diff
@@ -0,0 +1,472 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2018 Red Hat, Inc. and others.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""Copy containers between registries.
+
+This is a very limited version of the skopeo tool, but with support
+for manifests lists and OCI image indexes.
+https://github.com/containers/image/pull/400 will make this
+unnecessary.
+
+The only subcommand that is supported is 'copy', and the only supported image references
+are Docker registries of the form 'docker://docker-reference'.
+
+No global options are supported, and only selected options to 'copy' are supported (see
+--help for details.)
+
+Some other things that aren't implemented (but could be added if necessary):
+ - Handling of www-authenticate responses, necessary to log in to docker.io
+ - Special handling of 'docker.io' as 'registry-1.docker.io'
+ - Reading ~/.docker/config.json or $XDG_RUNTIME_DIR/containers/auth.json
+ - Handling foreign layers
+
+"""
+
+import click
+import json
+import logging
+import os
+import requests
+from requests.exceptions import SSLError, ConnectionError
+import shutil
+import tempfile
+from six.moves.urllib.parse import urlparse, urlunparse
```

Comment:
For [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports), let's regroup these imports by stdlib vs. third-party.